src/sass/_base.scss

```diff
@@ -1,5 +1,117 @@
 * {
   box-sizing: border-box;
+  font-family: "Jetbrains Mono", sans-serif;
 }
 
-body {}
+body {
+  background-color: $background;
+  margin: 0;
+}
+
+section {
+  border: 1px solid $button;
+  border-radius: 5px;
+  color: $light;
+  margin: 40px 10px;
+  padding: 5px;
+  text-align: center;
+
+  h2 {
+    margin: 0 auto;
+    width: 80%;
+  }
+
+  h3 {
+    margin: 10px auto;
+    width: 80%;
+  }
+
+  p {
+    margin: 40px auto;
+    padding: 0 20%;
+    width: 90%;
+  }
+
+  a {
+    color: $light;
+  }
+}
+
+.calendar {
+  margin: 0 auto;
+  text-align: left;
+  width: 150px;
+
+  .submit {
+    background-color: $button;
+    border: none;
+    border-radius: 10px;
+    color: #fff;
+    display: block;
+    font-size: 24px;
+    font-weight: 700;
+    margin: 0 auto;
+    margin: 20px;
```

Comment:
You have two `margin` rules back to back here, so the rule on line 53 is instantly overwritten.